packages/dcos-ui-service/buildinfo.json

```diff
@@ -1,9 +1,8 @@
 {
-  "single_source" : {
-    "kind": "git",
-    "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "211986ed6af9de08332adc696ceb20191e1aa36f",
-    "ref_origin": "master"
+  "single_source": {
+    "kind": "url_extract",
+    "url": "https://downloads.mesosphere.com/dcos-ui-update-service/latest/dcos-ui-update-service",
+    "sha1": "875cfce9-90ca-4174-8720-816b4cb7f10f"
```

Comment:
This is going to be a problem with the "latest" deployment method we chose

Reply:
@DanielMSchmidt Do you mean the `sha1`, because I am going to change it, or something else?

Reply:
Everytime we merge master we upload a new version to this exact URL. Therefore everytime we merge master the sha1 of the uploaded binary changes

Reply:
You are right, but I thought this was nevertheless the best method.

Reply:
yeah, it is. Just saying, there will be problems